reo/tests/test_critical_load_bau.py

```diff
@@ -58,20 +58,42 @@ def test_critical_load_bau_can_sustain_part_outage(self):
         nested_data = json.load(open(test_post, 'rb'))
 
         resp = self.get_response(data=nested_data)
-        self.assertHttpCreated(resp)
+        # self.assertHttpCreated(resp)
```

Comment:
Why is this line commented out?

Reply:
Commented out during testing and should have reverted